video-app/src/Utils/watchPlaylist.js

```diff
@@ -0,0 +1,5 @@
+const watchPlaylist = (video,id) =>{
```

Comment:
You can reuse `likePlaylist` function for this as well. as both of them doing the same operation. that is the purpose of utils. so that we can reuse them. Rename as `isVideoInList` and resuse it. Just a small suggestion

Reply:
will do that